docs/rfcs/bun-build-logs.ts

```diff
@@ -0,0 +1,40 @@
+export const enum MessageLevel {
+  err = 1,
+  warn = 2,
+  note = 3,
+  info = 4,
+  debug = 5,
+}
+
+export interface Location {
+  file: string;
+  namespace: string;
+  line: number;
+  column: number;
+  line_text: string;
```

Comment:
How about just `text`? to match with `MessageData#text` and also to avoid snake-case.